src/axom/multimat/multimat.hpp

```diff
@@ -216,6 +208,29 @@ class MultiMat
    */
   void setNumberOfCells(int num_cells);
 
+  /**
+   * \brief Sets the allocator ID to use for allocations.
+   *
+   * \param The allocator ID to use
+   */
+  void setAllocatorID(int alloc_id);
+
+  /**
+   * \brief Sets the allocator ID to use for Slam object allocations.
+   *  This should point to either a host Umpire pool if only being used on the
+   *  host, or a unified memory pool if also being used on the GPU.
+   *
+   * \param alloc_id the Umpire allocator ID to use
+   */
+  void setSlamAllocatorID(int alloc_id);
```

Comment:
Is there a reason that the allocator ID is not automatically the SLAM allocator?